api/webhook/post.go

```diff
@@ -405,6 +429,14 @@ func PostWebhook(c *gin.Context) {
 
 				return
 			}
+
+			l.WithFields(logrus.Fields{
+				"deployment":    d.GetNumber(),
```

Comment:
could do the same and set/reference `dr` from `UpdateDeployment` but not necessary

Reply:
yea, my thought was that if the update was successful, i don't need the new object. i think i used that approach consistently, but if not, i will update in those spots.